moz_security/sandboxes/heka/output/moz_security_ip_reputation.lua

```diff
@@ -0,0 +1,67 @@
+-- This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Also, I have no issues having the heavy hitters plugin output JSON instead of having to convert the tsv here.  Technically you don't even need the analysis plugin as you can do everything here (unless you want browse the output)

Reply:
The tsv output is useful for debugging, so I'm fine leaving it as tsv.  If we run into perf issues we could change the analysis plugin or change the server to accept tsv.